dev/intellij-setup.md

```diff
@@ -69,6 +69,14 @@ Before running or debugging the apps, you should do a `mvn clean install -Pdist
 
 You may also add `-Ddruid.console.skip=true` to the command if you're focusing on backend servers instead of frontend project. This option saves great building time.
 
+## Debugging Running Druid from Intellij
+Intellij IDEA debugger can attach to a local or remote Java process (Druid process). 
+First, you must enable debugging in the application.
```

Comment:
Would it be better to write these steps in a list format?

Something like:
```suggestion
Follow these steps to debug a Druid process using the IntelliJ IDEA debugger.
1. Enable debugging in the Druid process
   a. For Druid services (Overlord, Coordinator, etc.) ...
   b. For peons (...
2. Find the port assigned to the Druid process.
   a. For Druid services, you can find it in ...
   b. For peons, you can find the assigned ephemeral port ... task log
3. Create a remote configuration in the Run/Debug configurations dialog ...
4. Start (debug) the above remote configuration
```

Reply:
Done